src/checkstyle/checks/block/LeftCurlyCheck.hx

```diff
@@ -207,8 +207,8 @@ class LeftCurlyCheck extends Check {
 }
 
 typedef ParentToken = {
-	token:TokenTree,
```

Comment:
No nothing is manual here.. Ran reformat in intelliJ mainly to optimize imports.